components/admin_console/billing/plan_details/plan_details.tsx

```diff
@@ -0,0 +1,365 @@
+                                        />
+                                        {howBillingWorksLink}
+                                    </div>
+                                </Tooltip>
+                            )}
+                        >
+                            <i className='icon-information-outline'/>
+                        </OverlayTrigger>
+                    }
+                </div>
+            </div>
+            <div className='PlanDetails__subscriptionDate'>
+                <FormattedMessage
+                    id='admin.billing.subscription.planDetails.startDate'
+                    defaultMessage='Start Date: '
+                />
+                <FormattedDate
+                    value={startDate}
+                    day='numeric'
+                    month={getMonthLong(locale)}
+                    year='numeric'
+                />
+            </div>
+            <div className='PlanDetails__subscriptionDate'>
+                <FormattedMessage
+                    id='admin.billing.subscription.planDetails.endDate'
+                    defaultMessage='End Date: '
+                />
+                <FormattedDate
+                    value={endDate}
+                    day='numeric'
+                    month={getMonthLong(locale)}
+                    year='numeric'
+                />
+            </div>
+        </div>
+    );
+};
+
+export const planDetailsTopElements = (userCount: number, isPaidTier: boolean, isFreeTrial: boolean, userLimit: number) => {
+    let userCountDisplay = (
+        <div className='PlanDetails__userCount'>
+            <FormattedMarkdownMessage
+                id='admin.billing.subscription.planDetails.userCount'
+                defaultMessage='{userCount} users'
+                values={{userCount}}
+            />
+        </div>
+    );
+
+    if (!isPaidTier) {
+        userCountDisplay = (
+            <div
+                className={classNames('PlanDetails__userCount', {
+                    withinLimit: (userLimit - userCount) <= 5,
+                    overLimit: userCount > userLimit,
+                })}
+            >
+                <FormattedMarkdownMessage
+                    id='admin.billing.subscription.planDetails.userCountWithLimit'
+                    defaultMessage='{userCount} / {userLimit} users'
+                    values={{userCount, userLimit}}
+                />
+            </div>
+        );
+    }
+    let productName = (
+        <FormattedMessage
+            id='admin.billing.subscription.planDetails.productName.mmCloud'
+            defaultMessage='Mattermost Cloud'
+        />
+    );
+
+    if (isFreeTrial) {
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.cloudProfessional'
+                defaultMessage='Cloud Professional'
+            />
+        );
+    }
```

Comment:
We can change the name for all current plan to Cloud Professional so this condition is not needed

Reply:
ok